data/hai/hai missions.txt

```diff
@@ -1742,3 +1742,446 @@ mission "Nanachi Meeting"
+	label next
+	`	"When is the earliest that you could provide these ships to us?" Torsa's father asks.`
+	`	"It would only be a few weeks for us to have a number of ships ready, but I highly suggest that we wait until we have created hull repair technology. If these Unfettered Solifuges do not have such technology, then having it will give our ship an advantage, but we believe that we are still many years away from finishing it."`
+	`	An elder turns to you. "You have traveled many systems, <first> <last>. Do you know of any species who would be willing to provide the technology that we need?"`
+		goto branches
+	
+	label returning
+	`You return to <planet> and inform the elders that you have discovered a group that could help them with creating hull repair technology. They ask that you come to the council chambers in an hour while they find Torsa.`
+	`	"Now, tell us what it is you found," one of the elders says once you arrive.`
+	
+	label branches
+	branch both
+		has "remnant blood test pure"
+		has "Coalition: First Contact: done"
+	branch knowRemnant
+		has "remnant blood test pure"
+	branch knowCoalition
+		has "Coalition: First Contact: done"
+	
+	choice
+		`	"No. I don't know of any species that could help."`
+	`	"This is unfortunate. But you are a resourceful being. Perhaps you could explore the galaxy for a group able to help us." The elders and Torsa continue to discuss for a short while longer, deciding that Torsa and his team should create a small number of new ships to defend Cloudfire while you go searching for hull repair technology to finish the ship.`
+		accept
+	
+	label both
+	choice
+		`	"A group of humans in an area of space known as the Ember Waste have ships that are able to repair themselves."`
+			goto remnant
+		`	"There are a group of three alien species far from here known as the Coalition that have outfits able to repair a ship in flight."`
+			goto coalition
+	label knowRemnant
+	choice
+		`	"A group of humans in an area of space known as the Ember Waste have ships that are able to repair themselves."`
+			goto remnant
+	label knowCoalition
+	choice
+		`	"There are a group of three alien species far from here known as the Coalition that have outfits able to repair a ship in flight."`
+			goto coalition
+			
+	label remnant
+	apply
+		set "strider: remnant"
+	``
+		goto end
+	label coalition
+	apply
+		set "strider: coalition"
+	
+	label end
+	`	"Wonderful!" Torsa says. "This should turn many years of working into only a few months to finish our ship."`
+	`	The elders and Torsa continue to discuss for a short while longer, deciding that Torsa and his team should create a small number of new ships to defend Cloudfire while you travel to acquire hull repair technology to finish the ship. "We will put together a delegation for you to transport to these people you mention. They will be waiting in the spaceport shortly."`
+		decline
+
+mission "Strider: Remnant Hull Repair"
```

Comment:
I'm not entirely a fan of this mission. I understand why it is here, and it works. 

That being said, I'd much rather have an informational Remnant mission that the player can encounter regardless of any Hai missions, rather than have such a piece of info hidden away to only come up if the player triggers the Strider mission set first.